roles/agnosticv/templates/catalog_item.yml.j2

```diff
@@ -19,6 +19,9 @@ metadata:
 {%   for _label_k, _label_v in (_catalog.labels | default({})).items() %}
     babylon.gpte.redhat.com/{{ _label_k }}: {{ _label_v | to_json }}
 {%   endfor %}
+{%   if _stage in ['dev','test','prod'] %}
```

Comment:
Sometimes, stage is something else

```ilt.yaml
ocp-us-east-1.yaml
ocp-us-west-2.yaml
prod2.yaml
```

isn't `_stage` here the value of the `merged_vars.purpose` var instead ?

Reply:
The cases you mention are strange ones. I had them in mind when I restricted the list to dev/test/prod. Logically all of `ocp-us-east-1.yaml`, `ocp-us-west-2.yaml`, and `prod2.yaml` are production. We only have different "stages" there because we don't yet have variations of a base catalog item yet.

The `purpose` var in AgnosticV is used as simply a tag for cost reporting, and not used very consistently:
```
$ grep '^purpose:' -hr * | sed 's/ *#.*//' | sort | uniq -c
      7 purpose: apac-load-testing
    447 purpose: development
      1 purpose: develpment
     21 purpose: ilt
     35 purpose: prod
    324 purpose: production
      5 purpose: Production
      2 purpose: rhpds
      3 purpose: rhpds-development
     34 purpose: rhpds-prod
     33 purpose: rhpds-test
      1 purpose: RHPDS-Test
      1 purpose: summit2020
    218 purpose: test
      7 purpose: TEST
     38 purpose: testing
      1 purpose: Testing
```